src/components/Card.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+
+
+const Card = ({data}) => {
+  return (
+    <View style={styles.cardContainer}>
+      <Text style={styles.titleText}>{data.title}</Text>
+      <View style={styles.rowContainer}>
+        <Text style={styles.infoText}>{data.flats}</Text>
+        <Text style={styles.dot}>•</Text>
+        <Text style={styles.infoText}>{data.distance}</Text>
+      </View>
+      <View style={styles.infoContainer}>
```

Comment:
Create one InfoContainer component, where Min/Max and Amount can be passed, it will not duplicate code.

Reply:
Updated.